src/week-view.c

```diff
@@ -565,7 +565,30 @@ void week_view_add_calendar(WeekView* wv, Calendar* cal)
 	gtk_widget_queue_draw((GtkWidget*) wv);
 }
 
-int week_view_get_current_week(WeekView* wv)
+struct tm toLocaltime(time_t* timestamp)
```

Comment:
Please use snake case for function names. Also, local module functions should be `static`